src/shadowbox/docker/Dockerfile

```diff
@@ -12,8 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Newer node images have no valid content trust data.
+# Pin the image node:12.16.3-alpine (linux/amd64) by hash.
 # See versions at https://hub.docker.com/_/node/
```

Comment:
This link is more useful:
https://hub.docker.com/_/node?tab=tags&name=12-alpine

Reply:
Updated, with name=alpine, so we can see newer major version updates as well.